src/entt/entity/snapshot.hpp

```diff
@@ -55,7 +55,7 @@ class basic_snapshot {
 
         while(begin != last) {
             const auto entt = *(begin++);
-            ((reg->template all_of<Component>(entt) ? ++size[Index] : size[Index]), ...);
+            ((reg->template all_of<Component>(entt) ? ++size[Index] : 0), ...);
```

Comment:
Good catch! I've never ever hit this warning, really!
What about 0u though? So that we also avoid the integral promotion.

Reply:
Good idea, I'll fix that!